### PR TITLE
two resprites from nienhaus

### PR DESCRIPTION
**Pull request by pudl**
-resprites the toolbelt
-resprites glass cracks

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Ingame screenshots?

File: html/changelogs/pudl-resprites.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: pudl
```
Comment (Jordie0608) [inline html/changelogs/pudl-resprites.txt]: This needs to be a .yml, not a .txt

**Comment by dannno** [comment]
don't like the glass cracks tbh

**Comment by Miauw62** [comment]
We need screenies with more than just the toolbelt (typical assistant
loadout for example), and comparison pictures. With just the toolbelt it
looks like leather underwear.
Op 18-mrt.-2015 04:48 schreef "dannno" <notifications@github.com>:

**Comment by dannno** [comment]
yeah don't like them

**Comment by WJohn** [comment]
I like the toolbelt, but I can't say I like the cracked glass very much.

**Comment by Atskadan** [comment]
glass doesnt even make sense that isnt how glass breaks

**Comment by RemieRichards** [comment]
Not a fan of the glass.

**Comment by Nienhaus** [comment]
I think I know the problem with the glass. It was made slightly darker after after I made the cracks.

**Comment by Atskadan** [comment]
uh no the issue is the cracks are all over the place
when you hit glass you hit it in one spot and the damage spreads, like the current sprites
there is no way the glass would crack like you've made it unless you hit all over every time you strike it

**Comment by Nienhaus** [comment]
I was meaning more how it looks different on my topic.

**Comment by dannno** [comment]
simply not a fan of the glass, i think it should be left alone

**Comment by PKPenguin321** [comment]
hate to say it but yeah the glass isn't as good as what we have

**Pull request closed without merging**